src/08-time-tracking/time-tracking.apib

```diff
@@ -66,6 +66,7 @@ Get a list of tracked time.
                     + type: `workType` (string)
                     + id: `8a146eda-c9af-4b11-b63a-ddac47bb254d` (string)
                 + started_on: `2017-04-26` (string)
+                + is_duration_time_tracking: true (boolean)
```

Comment:
If we add it to the `.list` endpoint let's also add this to the `.info` endpoint.

Not sure a boolean is the best property to use here though. What about a type/...? Which is "duration" and not sure what the alternative would be?

Reply:
Not sure a boolean is the best property to use here though. What about a type/...? Which is "duration" and not sure what the alternative would be?

Reply:
Not sure a boolean is the best property to use here though. What about a type/...? Which is "duration" and not sure what the alternative would be?

Reply:
The other type is start-stop, currently our application is built around these 2 types, so I don't think another type will be added anytime soon

Reply:
After discussing with @mrtus, let's go for a `type` property with `start-stop` (?) and `duration`

Reply:
I will make it a type.